### PR TITLE
Curve staking

### PR DESCRIPTION
**Pull request by sparrowDom**
- adds curve staking section to stake page
- makes it possible to hide ogn staking section by clicking
- makes it possible to show/hide OGN/Curve staking by environmental variables
- adds brownie project to validate Curve gauge APY values

### CODE REVIEW

**Comment by sparrowDom** [comment]
This has been opened as a draft for some time, now it is ready to be reviewed @shahthepro

File: dapp/src/components/earn/CurveStake.js [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+  const [gaugeContract, setGaugeContract] = useState(false)
+  const [gaugeControllerContract, setGaugeControllerContract] = useState(false)
+  const chainId = useStoreState(ContractStore, (s) => s.chainId)
+  const readOnlyProvider = useStoreState(
+    ContractStore,
+    (s) => s.readOnlyProvider
+  )
+  const ognPrice = useStoreState(CoinStore, (s) => s.ogn.price)
+
+  const fetchBaseApy = async () => {
```
Comment (shahthepro) [inline dapp/src/components/earn/CurveStake.js:91]: nit: This function can be a React Hook to keep the code cleaner

Comment (sparrowDom) [reply dapp/src/components/earn/CurveStake.js]: Thanks good point, have separated that logic into a hook as suggested

**Comment by sparrowDom** [comment]
@shahthepro can you stamp this one please if there is nothing else to address?

**Review by shahthepro — APPROVED** [review]

**Comment by sparrowDom** [comment]
@shahthepro gracias :)

**Pull request merged**